.trunk/trunk.yaml

```diff
@@ -48,7 +48,7 @@ lint:
         - output: rewrite
           success_codes: [0]
           formatter: true
-          run: cairo-format ${target}
+          run: .venv/bin/cairo-format ${target}
```

Comment:
My bad for this. 
You should have cairo-format in your path in local in fact (symbolic link or installed in your system) and not modify this line.
Other solution to avoid failure,  is to modify the CI to create the .venv (hence uv/.venv would need to be setup) in the workflow trunk_check.yml as for now the cairo-lang is installed with pip.